torch/fx/node.py

```diff
@@ -650,7 +652,11 @@ def is_impure(self):
 
         # Check if an impure function.
         if self.op == "call_function":
-            return self.target in _side_effectful_functions
+            if self.target in _side_effectful_functions:
+                return True
+            if isinstance(self.target, torch._ops.OpOverload) and self.target._schema.is_mutable:
+                return True
```

Comment:
this avoid DCE'ing ops like `aten.dropout_.default`, which we should just check `is_mutable` for instead of relying on the `_side_effectful_functions` list.